test/appium/tests/critical/test_fallback.py

```diff
@@ -162,34 +163,89 @@ def _check_message(home_view, index):
-        key_pair_name = "New key pair"
-        key_pair_account_address = '0x' + account_to_add['address'].lower()
 
         wallet_1.just_fyi("Device 1: add a new regular account")
+        regular_account_name = "New regular account"
         regular_derivation_path = wallet_1.add_regular_account(account_name=regular_account_name)
-        regular_account_wallet_address = wallet_1.get_account_address().split(':')[-1]
+        regular_account_address = wallet_1.get_account_address().split(':')[-1]
         account_element = wallet_1.get_account_element(account_name=regular_account_name)
         wallet_1.close_account_button.click_until_presence_of_element(account_element)
 
-        wallet_1.just_fyi("Device 1: add a new key pair account")
+        if regular_account_address != expected_addresses[0]:
+            self.errors.append("Newly added regular account address %s doesn't match expected %s" % (
+                regular_account_address, expected_addresses[0]))
+        if regular_account_address not in expected_addresses:
+            self.errors.append("Newly added regular account address %s is not in the list of expected addresses %s" % (
+                regular_account_address, expected_addresses))
+
+        wallet_1.just_fyi("Device 1: add a new key pair account by importing recovery phrase")
+        account_element.swipe_left_on_element()
+        imported_key_pair_account_name = "Imported account"
+        imported_key_pair_name = "Imported key pair"
+        imported_key_pair_account_address = '0x' + account_to_add['address'].lower()
+        imported_key_pair_derivation_path = wallet_1.import_key_pair_using_recovery_phrase(
+            account_name=imported_key_pair_account_name,
+            passphrase=account_to_add['passphrase'],
+            key_pair_name=imported_key_pair_name)
+
+        wallet_1.close_account_button.click_until_presence_of_element(account_element)
+
+        wallet_1.just_fyi("Device 1: verify default and added key pairs details")
         account_element.swipe_left_on_element()
-        key_pair_derivation_path = wallet_1.add_key_pair_account(account_name=key_pair_account_name,
-                                                                 passphrase=account_to_add['passphrase'],
-                                                                 key_pair_name=key_pair_name)
+        wallet_1.get_account_element(account_name=imported_key_pair_account_name).swipe_left_on_element()
+        if not wallet_1.add_account_button.is_element_displayed():
+            wallet_1.get_account_element(account_name=imported_key_pair_account_name).swipe_left_on_element()
+        wallet_1.add_account_button.click()
+        wallet_1.create_account_button.click()
+        wallet_1.edit_key_pair_button.click()
+        expected_texts_regular = [
+            regular_account_name,
+            "eth:oeth:arb1:%s...%s" % (regular_account_address[:5], regular_account_address[-3:])
```

Comment:
I wouldn't test the full text with prefixes as we're planning to remove it soon, I believe just substring will be sufficient in this case